src/domain/thirdpartyRequests/transactions.ts

```diff
@@ -0,0 +1,177 @@
+ writing, the Mojaloop files are distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS
+ OF ANY KIND, either express or implied. See the License for the specific language governing
+ permissions and limitations under the License. Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file. Names of the original
+ copyright holders (individuals or organizations) should be listed with a '*' in the first column.
+ People who have contributed from an organization can be listed under the organization that actually
+ holds the copyright for their contributions (see the Gates Foundation organization for an example).
+ Those individuals should have their names indented and be marked with a '-'. Email address can be
+ added optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Sridhar Voruganti <sridhar.voruganti@modusbox.com>
+
+ --------------
+ ******/
+'use strict'
+
+import Hapi from '@hapi/hapi'
+import Logger from '@mojaloop/central-services-logger'
+import { Factory, ReformatFSPIOPError } from '@mojaloop/central-services-error-handling'
+import { Enum, Util } from '@mojaloop/central-services-shared'
+import { EventStateMetadata, EventStatusType } from '@mojaloop/event-sdk'
+import Mustache from 'mustache'
+import Config from '~/shared/config'
+import inspect from '~/shared/inspect'
+import { getStackOrInspect } from '~/shared/util'
+import * as types from '~/interface/types'
+
+/**
+ * Forwards POST transactions requests to destination FSP for processing
+ * @param {string} path Callback endpoint path
+ * @param {object} headers Headers object of the request 
+ * @param {string} method The http method POST
+ * @param {object} params Params object of the request 
+ * @param {object} payload Body of the POST request
+ * @param {object} span request span
+ * @throws {FSPIOPError} Will throw an error if no endpoint to forward the transactions requests is
+ * found, if there are network errors or if there is a bad response
+ * @returns {Promise<void>}
+ */
+async function forwardTransactionRequest(path: string, headers: Hapi.Util.Dictionary<string>, method: string, params: Hapi.Util.Dictionary<string>, payload?: types.ThirdPartyTransactionRequest, span?: any): Promise<void> {
+
+  const childSpan = span ? span.getChild('forwardTransactionRequest') : undefined
+  const fspiopSource: string = headers[Enum.Http.Headers.FSPIOP.SOURCE]
+  const fspiopDest: string = headers[Enum.Http.Headers.FSPIOP.DESTINATION]
+  const payloadLocal = payload || { transactionRequestId: params.ID }
+  const transactionRequestId: string = (payload && payload.transactionRequestId) || params.ID
+  const endpointType = Enum.EndPoints.FspEndpointTypes.THIRDPARTY_CALLBACK_URL_TRX_REQ_POST
+  let fspiopError: typeof Factory.FSPIOPError | undefined
+  let endpoint: string | undefined
```

Comment:
```suggestion
  let endpoint: string
```
Util.Endpoints.getEndpoint returns Promise<string>